README.md

```diff
@@ -85,5 +89,28 @@ SNP_id  gene_id chr_id  position        ref_allele      alt_allele      overall_
 10_100901494_G_T        ENSG00000107819 10      100901494       G       T       0.07303822937625754     7.707723138584862e-06   Na      Na      Na  Na       Na      6.790565417809094       eQTLGen-UBERON_0000178  Na      Na      Na      Na      [6.790565417809094]     ['eQTLGen-UBERON_0000178']  Na       Na      Na      Na
 ```
 
+### variant_disease output
+```
```

Comment:
Move here the description of output and the note about how gene are selected, before we give the example of the output

Reply:
I have added a description of the column of the output